README.md

```diff
@@ -36,5 +36,5 @@ about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq).
 * Submit Pull Request with comprehensive description of changes
 
 ## License
-The project is licensed under the terms of the Apache 2.0 license.
+The project is licensed under the terms of the Apache 2.0 license.<br/>
```

Comment:
This is not needed any more for the reason I have mentioned. Also, don't include html element in markdown file.